src/contracts/middleware/Middleware.sol

```diff
@@ -14,6 +14,7 @@
 // along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
 pragma solidity 0.8.25;
 
+import {console2} from "forge-std/console2.sol";
```

Comment:
We can take this off now :smile:

Reply:
Let's leave this one unresolved so we don't forget to take it out when it's ready